cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```
@@ -367,11 +370,16 @@ protected void onCreate(Bundle savedInstanceState) {
     createContent(savedInstanceState);
 
     videoSurfaceView = new VideoSurfaceView(this);
+
+    videoSurfaceView.setBackgroundColor(getThemeColorPrimary(this));
     a11yHelper = new CobaltA11yHelper(this, videoSurfaceView);
-    // Initialize videoSurfaceView as a single pixel to avoid rapid shifting
-    // between screen colors. Initializing with a width or height of 0 causes
-    // the call to addContentView to fail, so a single pixel is used.
-    addContentView(videoSurfaceView, new LayoutParams(1, 1));
+    addContentView(videoSurfaceView, new LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT));
+  }
+
+  private static int getThemeColorPrimary(final Context context) {
```

Can anything in this function throw an exception ? Seems unlikely, but if something can, maybe an exception catch and return a default value here.

---

Looking at the docs, neither of those function calls should throw an exception. I'm not sure if Theme can be returned as null but I added a check for that, as well as a check for the return value of resolveAttribute (will return false if it could not resolve).